modules/luci-base/po/zh-cn/base.po

```diff
@@ -3945,10 +3954,10 @@ msgid "unspecified"
 msgstr "未指定"
 
 msgid "unspecified -or- create:"
-msgstr "不指定或新建："
+msgstr "未指定 或 新建："
```

Comment:
I guess the original translation suits more the context.  The whitespace around "或" is superfluous and should be removed.

Reply:
The translation of "unspecified" should match the same word above. And the whitespace is used to highlight it's two actions, just like the original text "-or-", which puts two dashes around "or".

Reply:
In the context this text appears in, "unspecified" is actually an indication of choice of *action*, instead of *state*.  That's the subtle difference between "不指定" and "未指定".

It's conventional we write the sentence without whitespace between them, it will not cause any noticeable trouble at all.  It's overkill to manually add spaces here.

Reply:
So "unspecified" is actually an indication of choice of _state_, otherwise it will be "unspecify" (without past tense) here. "create", however, is an action (without past tense). It actually is this, leave the interface _unspecified_ to any firewall zone, or _create_ a new firewall zone. One is state, and another is action.

For the whitespace, I'm ok with or without it.

Reply:
"choice of state", "leave the interface unspecified" are still description about *action*, not state.

Reply:
"unspecif**ied**" _itself_ is a _state_, not "unspecif**y**"